From 32d4a140dc9a8d18604386ba08de911bef34c8f0 Mon Sep 17 00:00:00 2001
From: Jesse Keating <jkeating@redhat.com>
Date: Tue, 28 Jul 2009 20:06:30 -0700
Subject: [PATCH 2/2] Temporary workaround for accidentially re-downloading stale rpms.

This is causing us to download the same rpm over and over which
really doesn't work well.  This is a temporary work around given
to me by Mitr.
---
 src/bridge.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/bridge.py b/src/bridge.py
index 15f6422..32343ed 100644
--- a/src/bridge.py
+++ b/src/bridge.py
@@ -331,7 +331,7 @@ class SignRpmRequestType(RequestType):
         # attempts to initialize nss with our certificate database.
         import koji
 
-        if self.__koji_rpm_info is None:
+        if True:
             try:
                 d = {'name': self.__request_fields['rpm-name'],
                      'version': self.__request_fields['rpm-version'],
-- 
1.6.2.5

